definitions/divorce/json/FixedLists/FixedLists-nonprod.json

```diff
@@ -1,28 +1,28 @@
 [
```

Comment:
no, please leave it in the file it was.

Either or we will need to manually deal with it when release.

general email will go first nearly for sure. If not we will catch it. When we have some orphaned `nonprod` it may be accidentally forgotten.

This is a weakness of the feature toggle solution we have, but we must accept it.

Reply:
Ok